scripts/generate_solr_docs.py

```diff
@@ -91,6 +90,38 @@ def save_data(data, data_type=None):
 def variant_data(connection, limit=0):
     return variant.get_variant_data(connection, limit)
 
+def gene_data(connection, limit=0):
```

Comment:
Following the existing code design, the details of how to create the document_type _should_ be in a separate script to leave this one more general.

Reply:
Addressed. Now all gene related code was moved to `gene.py`.